docker/config/Caddyfile

```diff
@@ -50,6 +50,37 @@
     }
     rewrite @spa_router {http.matchers.file.relative}
 
+
+    # Proxy GET requests to VCR VC Service
+    @get_entities {
+        method GET
+        path /api/entities/*
+    }
+
+    route @get_entities {
+        {$API_RATE_LIMIT}
+        uri strip_prefix /api/entities
+        reverse_proxy {$VC_SERVICE_HOST}:{$VC_SERVICE_PORT} {
+            header_up Host {upstream_hostport}
+            header_up X-Forwarded-Host {host}
+        }
+    }
+
+    # Proxy POST requests to VCR VC Service
+    @post_entities {
+        method POST
+        path /api/entities/*
+    }
+
+    route @post_entities {
+        {$API_RATE_LIMIT}
+        uri strip_prefix /api/entities
+        reverse_proxy {$VC_SERVICE_HOST}:{$VC_SERVICE_PORT} {
+            header_up Host {upstream_hostport}
+            header_up X-Forwarded-Host {host}
+        }
+    }
+
```

Comment:
Why separate the `GET` and `SET` methods into separate proxy routes?  It should work with only one route that does not specify the method.

Also we'll need matching updates to the deployed version of the Caddy config over here; https://github.com/bcgov/orgbook-configurations/blob/main/openshift/templates/frontend/config/Caddyfile

Reply:
If you want to limit the requests to `GET` and `POST` you should be able to do it in one directive like:
```
   @get_post_entities {
        method GET POST
        path /api/entities/*
    }
```

Reply:
But then you're, perhaps obviously, excluding all other methods which might be over limiting.

Reply:
Maybe I'll just remove the method matcher. Right now, we're limiting methods like OPTIONS